python/package/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 3.5)
+project(bindings)
+
+# find_package(PCL REQUIRED)
+find_package(PCL COMPONENTS common REQUIRED)
+find_package(pybind11)
+
+# get the parent directory
+get_filename_component(PYTHON_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} DIRECTORY)
+
+# variable for holding folder name which contains pybind11 generated files
+set(PYBIND11_GEN pybind11-gen)
+
+# recursively get all pybind11 generated files
+file(GLOB_RECURSE PYBIND11_FILES ${PYTHON_DIRECTORY}/${PYBIND11_GEN}/*.cpp)
```

Comment:
How about creating a `CMakeLists.txt` along with the cpp files so there's no need of the devil glob? (maybe write as todo for future if not enough time otherwise)

Reply:
Why make a long CMakeLists? Why is glob "the devil" :joy: ?

Reply:
https://cliutils.gitlab.io/modern-cmake/chapters/intro/dodonot.html